crates/uv/src/main.rs

```diff
@@ -451,9 +451,20 @@ struct PipSyncArgs {
     ///   `python3.10` on Linux and macOS.
     /// - `python3.10` or `python.exe` looks for a binary with the given name in `PATH`.
     /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
-    #[clap(long, short, verbatim_doc_comment)]
+    #[clap(long, short, verbatim_doc_comment, conflicts_with = "system")]
     python: Option<String>,
 
+    /// Install packages into the system Python.
+    ///
+    /// By default, `uv` installs into the virtual environment in the current working directory or
+    /// any parent directory. The `--system` option instructs `uv` to instead use the first Python
+    /// found in the system `PATH`.
+    ///
+    /// `--system` is intended for use in continuous integration (CI) environments and should be
+    /// used with caution, as it can modify the system Python installation.
```

Comment:
Possibly over the top, but you could do something like this:

```suggestion
    /// WARNING: `--system` is intended for use in continuous integration (CI) environments.
    /// It should be used with caution, as it can modify the system Python installation.
```